docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -493,6 +493,8 @@ Query Parameters:
 -   **follow** – 1/True/true or 0/False/false, return stream. Default false
 -   **stdout** – 1/True/true or 0/False/false, show stdout log. Default false
 -   **stderr** – 1/True/true or 0/False/false, show stderr log. Default false
+-   **since** – UNIX timestamp (integer) to filter logs. Specifying a timestamp
```

Comment:
@duglin remote API only takes Unix timestamps, not date formats. That's the case with since/until parameters in /events as well. I kept it consistent.